Tests/DataPipeline/Support/Segment.swift

```diff
@@ -67,7 +67,7 @@ extension RawEvent {
         return nil
     }
 
-    var deviceAttributes: [String: Any]? {
+    var contextDeviceAttributes: [String: Any]? {
```

Comment:
Why did we rename this? 🤔

Reply:
to be able to differentiate between device attributes added as property vs device object in context